extensions/artemis-core/deployment/src/main/java/io/quarkus/artemis/core/deployment/ArtemisBuildTimeConfig.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkus.artemis.core.deployment;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "artemis", phase = ConfigPhase.BUILD_TIME)
+public class ArtemisBuildTimeConfig {
+
+    /**
+     * Whether or not an health check is published in case the smallrye-health extension is present (default to true).
```

Comment:
`(default to true)` can be removed since we automatically detect the value from ConfigItem during doc generation

Reply:
Fixed.